contracts/ProtocolReserve/ProtocolShareReserve.sol

```diff
@@ -210,7 +222,8 @@ contract ProtocolShareReserve is AccessControlledV8, ReentrancyGuardUpgradeable,
     function releaseFunds(address comptroller, address[] memory assets) external nonReentrant {
         _accruePrimeInterest();
 
-        for (uint256 i = 0; i < assets.length;) {
+        _ensureMaxLoops(assets.length);
```

Comment:
I don't think we need this check here. I think we should try to avoid before to have too many different assets for one Comptroller, but that could be risky because if we limit the number of assets we can lose some income. So, in this case, if there are many assets to release funds, maybe the sender of the `releaseFunds` could invoke the function in several transactions, splitting the array of assets